### PR TITLE
v1.8.2 Patch Update

### PR DESCRIPTION
**Pull request by theishshah**
Patch Release v1.8.2

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@ SHELL = /bin/bash
 # This value must be updated to the release tag of the most recent release, a change that must
 # occur in the release commit. IMAGE_VERSION will be removed once each subproject that uses this
 # version is moved to a separate repo and release process.
-export IMAGE_VERSION = v1.8.1
+export IMAGE_VERSION = 1.8.2
```
Comment (asmacdo) [inline Makefile:7]:
```suggestion
export IMAGE_VERSION = v1.8.2
```
I believe that will fix the broken link as well.

File: changelog/generated/1.8.2.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## v1.8.2
+
+### Changes
+
+- For helm-based operators, reverted the following PRs which modified helm release equality comparison. - https://github.com/operator-framework/operator-sdk/pull/5042 - https://github.com/operator-framework/operator-sdk/pull/4937. ([#5108](https://github.com/operator-framework/operator-sdk/pull/5108))
```
Comment (varshaprasad96) [inline changelog/generated/1.8.2.md:5]:
```suggestion
- For helm-based operators, reverted the following PRs which modified helm release equality comparison.
1. https://github.com/operator-framework/operator-sdk/pull/5042 
2. https://github.com/operator-framework/operator-sdk/pull/4937. ([#5108](https://github.com/operator-framework/operator-sdk/pull/5108))
```

File: website/config.toml [hunk 3]
```diff
@@ -95,6 +95,10 @@ algolia_docsearch = true
 
 ##RELEASE_ADDME##
 
+[[params.versions]]
```
Comment (varshaprasad96) [inline website/config.toml:98]: since release branch is already there, is this required to be added again?
cc: @asmacdo

File: Makefile (hunk 1 above)
Comment (rashmigottipati) [reply Makefile:7]: The same replacement (1.8.2 --> v1.8.2) needs to happen across all files.

File: website/config.toml (hunk 3 above)
Comment (rashmigottipati) [reply website/config.toml:98]: I don't think we should be adding this as the release branch is already there and this is just a patch release.

**Pull request closed without merging**